src/components/regular-pop-up-menu/RegularPopUpMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { ModalHeader } from "..";
```

Comment:
@cris22380 avoid these kind of imports. This causes circular references, use named imports